### PR TITLE
feat: add curvature

### PR DESCRIPTION
**Pull request by axelstr**

### CODE REVIEW

File: curvature.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	if sNoUnit != strings.TrimSpace(sNoUnit) {
+		return fmt.Errorf("unmarshal curvature: input '%s' has invalid whitespace", s)
+	}
+	magnitude, err := strconv.ParseFloat(sNoUnit, 64)
+	if err != nil {
+		return fmt.Errorf("unmarshal curvature: parse '%s': %w", s, err)
+	}
+	*c = Curvature(magnitude)
+	return nil
+}
```
Comment (andersson-robin) [inline curvature.go:47]: This pattern is a bit different compared to UnmarshalStrings in other packages. Just curious why that is?

**Review by andersson-robin — COMMENTED** [review]
LGTM! Only have a question in my other comment.

File: curvature.go (hunk 1 above)
Comment (axelstr) [reply curvature.go:47]: - The default unmarshler didn't like the `/` sign.
- The default unmarshler supports unit prefixes like kilo, centi etc. In our case that would just be confusing so we shouldn't support anything other than `/m` at the moment. For example, one wouldn't write `1k/m=1000/m`

Comment (andersson-robin) [reply curvature.go:47]: Makes sense, thanks for clarifying! :+1:

**Review by andersson-robin — APPROVED** [review]
:1st_place_medal:

**Review by felixmr1 — APPROVED** [review]
LGTM

**Pull request merged**